src/PKSim.Core/Services/IndividualProteinFactory.cs

```diff
@@ -35,15 +35,15 @@ public override IndividualMolecule AddMoleculeTo(ISimulationSubject simulationSu
 
          AddGlobalExpression(molecule,
             RelExpParam(REL_EXP_BLOOD_CELLS),
-            FractionParam(FRACTION_EXPRESSED_BLOOD_CELLS, CoreConstants.Rate.ZERO_RATE),
+            FractionParam(FRACTION_EXPRESSED_BLOOD_CELLS, CoreConstants.Rate.ONE_RATE),
```

Comment:
We are not using the DB For this. I would love to update that to make sure we are relying only one one source of truth. I might get a stab at it in the next PR